### PR TITLE
Apple page clone

### PR DESCRIPTION
**Pull request by eduardoreisalvarenga**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+    </div><!--Navbar Items End-->
+    <div class="main-text-header"> <!--Container which contains my header-->
+    </div>
+    <div class="secondary-text"><!--Container for my secondayr text-->
+    </div>
+    <div class="your-verse"> <!--Container for the last text inside main-->
+    </div>
+  </main><!--Container end-->
+  <div class="flex-container"><!--Flex Container for secundary options start-->
```
Comment (Torres-ssf) [inline index.html:35]: [OPTIONAL]
You did a nice job using HTML5 semantic tag. You can also make this one be a `section` tag instead

**Review by Torres-ssf — APPROVED** [review]
### Wow :clap: Nicely Done :1st_place_medal: :fireworks: :tada: 
Congratulations, your project looks great. You got everything right.
I'm approving this project, go on and merge to master
Good luck with the next one :smile: 
Happy coding :computer:

File: index.html (hunk 1 above)
Comment (eduardoreisalvarenga) [reply index.html:35]: Okay. Thanks for your review! XD

**Pull request merged**